docassemble/juvenilesealing/data/static/gbls.css

```diff
@@ -186,8 +186,8 @@ button.btn-secondary:disabled,
 .btn-secondary:not(:disabled):not(.disabled).active,
 .btn-secondary:not(:disabled):not(.disabled):active,
 .show>.btn-secondary.dropdown-toggle {
-  background-color: rgba( 99, 99, 99, 1 );
-  border-color: rgba( 99, 99, 99, 1 );
+  background-color: rgba( 70, 70, 70, 1 );
+  border-color: rgba( 70, 70, 70, 1 );
   background-color: white;
```

Comment:
repeated background-color and border-color. The later style will actually take precedence so the new style wouldn't be getting applied

Reply:
Also be careful here - the first background-color is dark gray while the other is white